src/api.js

```diff
@@ -0,0 +1,27 @@
+const API_URL = 'https://bootcamp-api.codeit.kr';
+
+export async function fetchProfileData() {
```

Comment:
보통 우리가 `OO아, 너 프로필좀 알려줘.`하는 것처럼 어떤 명사(여기서는 프로필) 자체가 데이터라는 정보도 같이 포함하게 되는데요! 그래서 보통 `OO아, 너 프로필 데이터좀 알려줘.`라고 하지는 않는 것 처럼요! (약간 억지지만 요런 느낌으로 봐주시면 될 것 같아요~)

그래서 코드를 작성할때도 -Data는 제거하고 `fetchProfile`로 표현해주셔도 충분할 것 같아요~
```suggestion
export async function fetchProfile() {
```

Reply:
이 부분은 api를 호출하는 함수에 한정된 내용인데요!

브라우저가 서버랑 통신을 하기 위해서는 `http`라는 대화 방식을 통해서 통신을 하게 되고 이 과정에서 브라우저가 서버에게 원하는 행위를 표현하기 위해 `get`과 같은 `http method`를 서버에 전달하게 되는데요! 주로 많이 사용하는 http method에는 아래와 같은 것들이 있어요!
- get
- post
- put
- patch
- delete

api 함수는 어떤 http method를 사용하냐를 통해서 그 동작을 더욱 명확하게 알 수 있기 때문에 함수명의 앞에다가 사용하는 http method 이름을 붙여주면 더 좋을 것 같은데요~
다만 지금도 충분히 좋고 하나의 의견으로 생각해주시면 좋을 것 같아요!
(요런 리뷰들은 반영해도 좋고 안해도 좋은 리뷰입니다~)
```suggestion
export async function getProfileData() {
```